### PR TITLE
docs: add sovereign landing zone

### PR DESCRIPTION
**Pull request by jaredfholgate**
# Pull Request

## Description

Docs updates for Sovereign Landing Zone

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the projects associated license.

### CODE REVIEW

**Review by VeronicaSea — APPROVED** [review]
Looks good to me.

**Review by VeronicaSea — DISMISSED** [review]

File: docs/wiki/[User-Guide]-Quick-Start-Phase-2-Azure-DevOps.md [hunk 1]
```diff
@@ -144,12 +146,15 @@ Although you can just run `Deploy-Accelerator` and fill out the prompted inputs,
...
     | `apply_approvers` | `TF_VAR` | `<email-address>` | This is a list of service principal names (SPN) of people you wish to be in the group that approves apply of the Azure landing zone module. This is an array of strings like `["abc@xyz.com", "def@xyz.com", "ghi@xyz.com"]`. You may need to check what the SPN is prior to filling this out as it can vary based on identity provider. Use empty array `[]` to disable approvals. Note if supplying via the user interface, use a comma separated string like `abc@xyz.com,def@xyz.com,ghi@xyz.com`. |
     | `create_branch_policies` | `TF_VAR` | `true` | This controls whether to create branch policies for the repository. This defaults to `true`. |
+    | `architecture_definition_name` | `TF_VAR` | N/A | This is the name of the architecture definition to use when applying the ALZ archetypes via the architecture definition template. This is only relevant to starter modules used by Microsoft Cloud for Industry, such as the `sovereign_landing_zone` starter module. This defaults to `null`. |
```
Comment (azure-lm) [inline docs/wiki/[User-Guide]-Quick-Start-Phase-2-Azure-DevOps.md:149]: I know we pulled the "Microsoft Cloud for Sovereignty" language out in favor of "Sovereign Landing Zone". Is a customer coming here first going to have any context for "Microsoft Cloud for Industry" or should we drop this part of the description?

File: docs/wiki/[User-Guide]-Starter-Module-Terraform-Sovereign-Landing-Zone.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+Example input files can be found here:
+
+- [inputs-azure-devops-terraform-sovereign-landing-zone.yaml][example_powershell_inputs_azure_devops_terraform_sovereign_landing_zone]
+- [inputs-github-terraform-sovereign-landing-zone.yaml][example_powershell_inputs_github_terraform_sovereign_landing_zone]
+- [inputs-local-terraform-sovereign-landing-zone.yaml][example_powershell_inputs_local_terraform_sovereign_landing_zone]
+
+The following table describes the inputs for the `sovereign_landing_zone` starter module.
+
+| Input | Required | Type | Default Value | Description |
```
Comment (azure-lm) [inline docs/wiki/[User-Guide]-Starter-Module-Terraform-Sovereign-Landing-Zone.md:15]: This looks like the comprehensive list of inputs. Originally, this had just the required ones, but should this be all of them? Some of the optional inputs have descriptions that call out to other sections that were only included in the accelerator doc. So, if we should have the full list here, we should also pull over the other sections.

File: docs/wiki/examples/powershell-inputs/inputs-local-terraform-sovereign-landing-zone.yaml [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+bootstrap_location: "<region-1>"
+starter_locations: ["<region-1>"]  # NOTE: SLZ only support a single region by design
+root_parent_management_group_id: ""
+subscription_id_management: "<management-subscription-id>"
+subscription_id_identity: "<identity-subscription-id>"
+subscription_id_connectivity: "<connectivity-subscription-id>"
+
+# Bootstrap Inputs
+target_directory: ""
+create_bootstrap_resources_in_azure: true
```
Comment (VeronicaSea) [inline docs/wiki/examples/powershell-inputs/inputs-local-terraform-sovereign-landing-zone.yaml:20]: Could we set this create_bootstrap_resources_in_azure: false ?

Comment (VeronicaSea) [reply docs/wiki/examples/powershell-inputs/inputs-local-terraform-sovereign-landing-zone.yaml]: Set it to false

**Review by VeronicaSea — DISMISSED** [review]

**Review by VeronicaSea — APPROVED** [review]

**Pull request merged**